src/app/dim-ui/Sheet.m.scss

```diff
@@ -18,3 +18,7 @@
     opacity: 0.6;
   }
 }
+
+.noHeader {
+  right: 16px;
+}
```

Comment:
This doesn't quite seem to work, e.g. in the armory sheet 

![grafik](https://github.com/DestinyItemManager/DIM/assets/14299449/ac0d8060-2d2f-4b5c-8e43-2111e5cab0a7)

Reply:
Ah, this is what I get for pulling these changes into PRs out of order...